src/js/bootstrap-slider.js

```diff
@@ -1148,6 +1148,9 @@ const windowIsDefined = (typeof window === "object");
 
 				this.handle1.style[this.stylePos] = positionPercentages[0]+'%';
 				this.handle1.setAttribute('aria-valuenow', this._state.value[0]);
+				if (isNaN(this.options.formatter(this._state.value[0])) ) {
+					this.handle1.setAttribute('aria-valuetext', this.options.formatter(this._state.value[0]));
```

Comment:
could you also add a section to our `README.md` that documents this new ARIA property and add an example to our `/tpl/index.tpl` page?